HelpSource/Classes/DrawGridX.schelp

```diff
@@ -0,0 +1,28 @@
+CLASS:: DrawGridX
+summary:: A sub-method of DrawGrid, used for setting up its horizontal size
+categories:: GUI>Accessories
+related:: Reference/plot, Classes/GridLines, Classes/Plotter, Classes/UserView, Classes/DrawGrid
```

Comment:
I just checked, Reference/plot should be fine -- there is a plot.schelp file. Sorry for noise.